bindata/manifests/metrics-exporter/metrics-config-map.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: sriov-network-metrics-exporter-config
+  namespace: {{.Namespace}}
+data:
+  drivers.yaml: |-
```

Comment:
we can remove this, since we plan to remove the file from metrics exporter right?

Reply:
I  would leave this configuration until we reach a consensus on that configmap. Setting the version to 0.0.0 enables the specific driver on any version. WDYT?

Reply:
sure that's fine with me